From 893c45517b09752bbcef281c29de9bcb52f3e122 Mon Sep 17 00:00:00 2001
From: Zhang Bo <bo.zhang@nxp.com>
Date: Wed, 22 Mar 2017 12:04:32 +0800
Subject: [PATCH 1680/5242] MA-9371-2[Android_MX7ULP-EVK] change the dts file
 to solve the touch point not match display issue

commit  d9ad174098eb4dd7dd667da44bbb3cccc3622d46 from
https://source.codeaurora.org/external/imx/linux-imx.git

The default touch panel parameters in the dts file is used for the maximum supported display resolution.
It needs to calibrate the touch panel to match other resolution display screen.
The display screen for mx7ulp-evk is not matched the default parameters.

change the dts file, so it need not to calibrate the touch panel for mx7ulp-evk board.

Change-Id: I24b1ceeef7f584b6ddf057794271dfa3a5875c0b
Signed-off-by: Zhang Bo <bo.zhang@nxp.com>
Signed-off-by: Gao Pan <pandy.gao@nxp.com>
(cherry-pick from 670d180d907c9bada3972ac9aded51a1becaf646)
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx7ulp-evk.dts |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/imx7ulp-evk.dts b/arch/arm/boot/dts/imx7ulp-evk.dts
index 019ad99..cd87a87 100644
--- a/arch/arm/boot/dts/imx7ulp-evk.dts
+++ b/arch/arm/boot/dts/imx7ulp-evk.dts
@@ -400,7 +400,7 @@
 		focaltech,reset-gpio = <&gpio3 1 0x01>;
 		focaltech,irq-gpio = <&gpio3 0 0x02>;
 		focaltech,max-touch-number = <5>;
-		focaltech,display-coords =  <0 0 1080 1920>;
+		focaltech,display-coords =  <0 0 272 480>;
 
 		focaltech,have-key;
 		focaltech,key-number = <3>;
-- 
1.7.9.5

